tests/test_adapt.py

```diff
@@ -233,3 +236,30 @@ def test_inline_model_custom_node_nested(old_squeeze: onnx.ModelProto):
     # Add another node to the model to trigger the adaption logic
     c = op18.identity(b)
     build({"a": a}, {"c": c})
+
+
+def test_if_adapatation_squeeze():
+    cond = argument(Tensor(numpy.bool_, ()))
+    b = argument(Tensor(numpy.float32, (1,)))
+    squeezed = Squeeze11.squeeze11(b, [0])
+    out = op18.if_(
+        cond,
+        then_branch=lambda: [squeezed],
+        else_branch=lambda: [Squeeze11.squeeze11(b, [0])],
+    )
+    model = build({"b": b, "cond": cond}, {"out": out[0]})
+
+    # predict on model
+    b = numpy.array([1.1], dtype=numpy.float32)
+    cond = numpy.array(True, dtype=numpy.bool_)
+    out = ort.InferenceSession(model.SerializeToString()).run(
+        None, {"b": b, "cond": cond}
+    )
+
+
+def test_if_adaptation_const():
+    sq = op19.const(1.1453, dtype=numpy.float32)
+    b = argument(Tensor(numpy.float32, ("N",)))
+    cond = op18.equal(sq, b)
+    out = op18.if_(cond, then_branch=lambda: [sq], else_branch=lambda: [sq])
+    build({"b": b}, {"out": out[0]})
```

Comment:
Can you check the opset version of the `"ai.onnx"` aka `""` of the built artifact please? It should be larger than 11.

Reply:
Yes, can do